womtool/src/test/resources/validate/wdl_draft2/valid/relative_local_import/relative_local_import.wdl

```diff
@@ -0,0 +1,11 @@
+import "./womtool/src/test/resources/validate/wdl_draft2/valid/task_only/task_only.wdl" as imports
```

Comment:
I suspect this would have worked regardless of the change you made, since it was incorrectly validating that the WDL being imported was in a sub-directory of the current WDL's location.

I think making the path relative to the current file would be a good test since it lets you backtrack a little, eg
```
import ../task_only/task_only.wdl
```

Reply:
I tried that, but it fails because the tool is looking for files relative to womtool rather than the WDL itself. I'm just restoring the previous behaviour. I've verified that the new test actually tests `localFileResolver` through the debugger. Should I make it so that WOMtool looks for files relative to the WDL file?

Reply:
Oh right, because the `localFileResolver` is fixed relative to PWD...

On consideration I think the best answer is to give the DirectoryResolver a `checkWhetherThisFileIsInASubdirectory: Boolean` parameter so that womtool can allow escaping their directory but server mode cannot